scripts/segmentation/test.py

```diff
@@ -68,14 +68,14 @@ def test(args):
             im_paths = dsts
             predicts = evaluator.parallel_forward(data)
             for predict, impath in zip(predicts, im_paths):
-                predict = mx.nd.squeeze(mx.nd.argmax(predict[0], 1)).asnumpy() + \
+                predict = mx.nd.squeeze(mx.nd.argmax(predict[0], 0)).asnumpy() + \
```

Comment:
Following the standard practice, we use the first dimension as Batch dimension by default. Could you please still use dim=1.

Reply:
Fixed it. : )